source/includes/concept/_widget.md

```diff
@@ -0,0 +1,17 @@
+## Widget
+
+You have fetched your data from your [dataset](#dataset) using [queries](#query). And you know how to build custom visualizations for geospatial data using [layers](#layer). Yet, sometimes your data is not geo-located, and you want to build a plain graphical representation of it: this is when **widgets** come in handy.
```

Comment:
You can still use widgets for geo-referenced data - should clarify that it can be used for any kind of data, georeferenced or not.

Reply:
Clarified, thanks for pointing out :+1: